File: Assets/AltUnityTester/AltUnityDriver/Commands/OldFindObjects/WaitForElementToNotBePresent.cs
public class WaitForElementToNotBePresentDriver : AltBaseCommand
{
    string name;
    string cameraName;
    bool enabled;
    double timeout;
    double interval;
    public WaitForElementToNotBePresentDriver(SocketSettings socketSettings, string name, string cameraName, bool enabled, double timeout, double interval) : base(socketSettings)
    {
        this.name = name;
        this.cameraName = cameraName;
        this.enabled = enabled;
        this.timeout = timeout;
        this.interval = interval;
    }
    public void Execute()
    {
        double time = 0;
        bool found = false;
        AltUnityObject altElement = null;
        while (time <= timeout)
        {
            found = false;
            try
            {
                altElement = new FindElementDriver(SocketSettings, name, cameraName, enabled).Execute();
                found = true;
                System.Threading.Thread.Sleep(System.Convert.ToInt32(interval * 1000));
                time += interval;
                UnityEngine.Debug.Log("Waiting for element " + name + " to not be present");
            }
            catch (System.Exception)
            {
                break;
            }

        }
        if (found)
            throw new Assets.AltUnityTester.AltUnityDriver.WaitTimeOutException("Element " + name + " still found after " + timeout + " seconds");
    }
}

File: Assets/AltUnityTester-BindingsAndExamples(can_be_deleted)/Bindings~/python/altunityrunner/commands/OldFindObjects/wait_for_element_to_not_be_present.py
from altunityrunner.commands.command_returning_alt_elements import CommandReturningAltElements
from altunityrunner.commands.OldFindObjects.find_element import FindElement
from altunityrunner.altUnityExceptions import WaitTimeOutException
import time
class WaitForElementToNotBePresent(CommandReturningAltElements):
    def __init__(self, socket,request_separator,request_end,appium_driver,value,camera_name='', timeout=20, interval=0.5,enabled=True):
        super().__init__(socket,request_separator,request_end,appium_driver)
        self.value=value
        self.camera_name=camera_name
        self.timeout=timeout
        self.interval=interval
        self.enabled=enabled
    
    def execute(self):
        t = 0
        while (t <= self.timeout):
            try:
                print('Waiting for element ' + self.value + ' to not be present...')
                alt_element=FindElement(self.socket,self.request_separator,self.request_end,self.appium_driver,self.value,self.camera_name,self.enabled).execute()
                time.sleep(self.interval)
                t += self.interval
            except Exception:
                break
        if t>=self.timeout:
            raise WaitTimeOutException('Element ' + self.value + ' still found after ' + str(self.timeout) + ' seconds')